### ISSUE TITLE
Sky Crash Factory Landing Map

### ISSUE THREAD

**Issue by Rudy2D2** [DarkPacks/SkyFactory-4#1292]
## Bug Report
When I dig to the bottom of the map (Sky Crash Factory Landing), I fall into the void and the game freezes. There does not seem to be bedrock or another block that gravity does not affect at the lower Y levels. Maybe this is causing all of the dust (literally the whole world) to fall into the void and lock up the game?

## Expected Behaviour
One should hit a non-falling block at lowest Y levels. The game should not freeze if one falls into the void, if one should fall into the void at all on this preset.

## Possible Solution
Add a layer of bedrock or some other block to the bottom of the map.

## Steps to Reproduce (for bugs)
1. Load a Sky Crash Factory Landing map.
2. Dig to Y level 0.
3. Shut down the game through task manager as game will no longer respond.

## Logs
No logs as game does not actually crash.

## World Information
* Preset: Sky Crash Factory Landing
* Prestige: No, but it shouldn't matter.
* Modpack Version world created in: Last Released
* Additional Content Installed: None

## Client Information

* Modpack Version: Newest
* Java Version: Whatever Twith launcher uses. I update my personal Java regularly.
* Launcher Used: Twitch
* Memory Allocated: 5+ GB
* Server/LAN/Single Player: Single Player
* Optifine Installed: No
* Shaders Enabled: No

**Comment by sam-kirby** [comment]
A lower bedrock layer should be present.

https://github.com/DarkPacks/SkyFactory-4/blob/master/src/config/topography/SF4/sky_crash_factory_landing.txt#L6

**Comment by Rudy2D2** [comment]
It is not, though; I am sure of that now. So I tried to change the values in the level generator section, but nothing seemed to happen. Then I removed the other lines, and it worked. So by process of elimination, I found that the dune generator at lines 24 and 25 is overwriting the bedrock level:
     dune = new DuneTestGenerator(<contenttweaker:dust>)
     addGenerator(dune)
I don't know how Topography works, but maybe some other parameters are needed in the DuneTestGenerator to avoid removing the bedrock? Maybe a Y level specification?
